### ISSUE TITLE
Error "Unknown provider NotificationCenter" prevents Kibana from starting

### ISSUE THREAD

**Issue by rigon** [sw-jung/kibana_notification_center#8]
I am running ElasticSearch and Kibana from docker, version 6.1.1.
Here is the list of commands:

    docker run -it -e discovery.type=single-node --name elasticsearch -p 9200:9200 -p 9300:9300 docker.elastic.co/elasticsearch/elasticsearch-oss:6.1.1
    docker run -it -p 5601:5601 --link elasticsearch:elasticsearch docker.elastic.co/kibana/kibana-oss:6.1.1

Then, I connect to Kibana container and run:

    kibana-plugin install https://github.com/sw-jung/kibana_notification_center/releases/download/v6.1.1/notification_center-6.1.1.zip

But then Kibana gives an error which prevents it from starting:

    Version: 6.1.1
    Build: 16350

    Error: Uncaught Error: [$injector:unpr] Unknown provider: clusterUuidProvider <- clusterUuid <- NotificationCenter
    http://errors.angularjs.org/1.6.5/$injector/unpr?p0=clusterUuidProvider%20%3C-%20clusterUuid%20%3C-%20NotificationCenter (http://localhost:5601/bundles/commons.bundle.js?v=16350:29)
        at window.onerror (http://localhost:5601/bundles/commons.bundle.js?v=16350:45:8729)

I tried also with a couple of 5.x versions, but I got the same error.

Am I missing something or is there any bug with the plugin?

**Comment by sw-jung** [comment]
This plugin has never been tested with docker. 😢

Have you ever had any other changes to kibana?(e.g. kibana.yml, install other plugins, ...)

I'll try to resolve it quickly!

**Comment by rigon** [comment]
No, it is just image as is, downloaded from elastic.co. Then I just installed the plugin.
You can reproduce the error with the commands I provided.

**Comment by sw-jung** [comment]
@rigon Sorry for delayed update. Please use v6.1.2 or higher! 😄

**Comment by rigon** [comment]
It works like a charm!
Sorry, I missed your notification in the middle of others...
